src/components/PersonalSettings.vue

```diff
@@ -39,6 +39,15 @@
 					{{ t('assistant', 'No suitable providers are available. They must first be enabled by your administrator.') }}
 				</span>
 			</div>
+			<div v-else>
+				<h3>{{ t('assistant', 'Configured Backends') }}</h3>
+				<p>{{ t('assistant', 'The following services are used as backends for Nextcloud Assistant:') }}</p>
+				<ul>
+					<li v-for="(providerName, taskName) in providers" :key="taskName">
+						{{ t('assistant', '{providerName} for {taskName}', {providerName, taskName}) }}
```

Comment:
```suggestion
						{{ t('assistant', '{providerName} for {taskName}', { providerName, taskName }, null, { escape: false }) }}
```
Otherwise simple quotes are escaped and we get this:
![image](https://github.com/user-attachments/assets/70fb4737-8a49-4907-bbfb-79635185a955)